code/game/machinery/Sleeper.dm

```diff
@@ -39,62 +39,60 @@
-	if(..())
+/obj/machinery/sleep_console/interact(var/mob/user)
+	if(!connected)
 		return
-	if (src.connected)
-		var/mob/living/occupant = src.connected.occupant
-		var/dat = list()
-		if(connected.on)
-			dat += "<font color='blue'><B>Performing anaesthesic emergence...</B></font>" //Best I could come up with
-			dat += "<HR><A href='?src=\ref[src];toggle_autoeject=1'>Auto-eject occupant: [connected.auto_eject_after ? "Yes" : "No"]</A><BR>"
-		else
-			dat += "<font color='blue'><B>Occupant Statistics:</B></FONT><BR>"
-			if (occupant)
-				var/t1
-				switch(occupant.stat)
-					if(0)
-						t1 = "Conscious"
-					if(1)
-						t1 = "<font color='blue'>Unconscious</font>"
-					if(2)
-						t1 = "<font color='red'>*dead*</font>"
-					else
-				dat += text("[]\tHealth %: [] ([])</FONT><BR>", (occupant.health > 50 ? "<font color='blue'>" : "<font color='red'>"), occupant.health, t1)
-				if(iscarbon(occupant))
-					var/mob/living/carbon/C = occupant
-					dat += text("[]\t-Pulse, bpm: []</FONT><BR>", (C.pulse == PULSE_NONE || C.pulse == PULSE_THREADY ? "<font color='red'>" : "<font color='blue'>"), C.get_pulse(GETPULSE_TOOL))
-				dat += text("[]\t-Brute Damage %: []</FONT><BR>", (occupant.getBruteLoss() < 60 ? "<font color='blue'>" : "<font color='red'>"), occupant.getBruteLoss())
-				dat += text("[]\t-Respiratory Damage %: []</FONT><BR>", (occupant.getOxyLoss() < 60 ? "<font color='blue'>" : "<font color='red'>"), occupant.getOxyLoss())
-				dat += text("[]\t-Toxin Content %: []</FONT><BR>", (occupant.getToxLoss() < 60 ? "<font color='blue'>" : "<font color='red'>"), occupant.getToxLoss())
-				dat += text("[]\t-Burn Severity %: []</FONT><BR>", (occupant.getFireLoss() < 60 ? "<font color='blue'>" : "<font color='red'>"), occupant.getFireLoss())
-				var/sleepytime = max(occupant.paralysis, occupant.sleeping)
-				dat += text("<HR>Paralysis Summary %: [] ([] seconds left!)<BR>", sleepytime, round(sleepytime*2))
-				dat += "<a href ='?src=\ref[src];wakeup=1'>Begin Wake-Up Cycle</a><br>"
-				if(occupant.reagents)
-					for(var/chemical in connected.available_options)
-						dat += "[connected.available_options[chemical]]: [occupant.reagents.get_reagent_amount(chemical)] units<br>"
-				dat += "<HR><A href='?src=\ref[src];refresh=1'>Refresh meter readings each second</A><BR>"
+	var/mob/living/occupant = connected.occupant
+	var/dat = list()
+	if(connected.on)
+		dat += "<B>Performing anaesthesic emergence...</B>" //Best I could come up with
+		dat += "<HR><A href='?src=\ref[src];toggle_autoeject=1'>Auto-eject occupant: [connected.auto_eject_after ? "Yes" : "No"]</A><BR>"
+	else
+		dat += "<b>Occupant statistics:</b><BR>"
+		if (occupant)
+			var/occupant_status = "???"
+			switch(occupant.stat)
+				if(CONSCIOUS)
+					occupant_status = "conscious"
+				if(UNCONSCIOUS)
+					occupant_status = "<span class='average'>unconscious</span>"
+				if(DEAD)
+					occupant_status = "<span class='average'>*dead*</span>"
```

Comment:
`<span class='warning'>` would probably do what you're looking for

Reply:
Isn't that a goonchat thing?

Reply:
It is.